slackbot.go

```diff
@@ -173,6 +268,9 @@ func (s *SlackBot) GetChannelByName(name string) SlackChannel {
 	if strings.HasPrefix(name, "G") {
 		return s.groups[name]
 	} else {
+		if strings.HasPrefix(name, "D") {
```

Comment:
i tried to keep it consistent with your existing style, but we can break it out to be more idiomatic.